language-plutus-core/src/Crypto.hs

```diff
@@ -0,0 +1,13 @@
+module Crypto (verifySignature) where
+
+import           Crypto.ECC.Ed25519Donna
+import           Crypto.Error            (maybeCryptoError)
+import qualified Data.ByteString.Lazy    as BSL
+import           PlutusPrelude
+
+-- we default to 'False' in accordance with the spec
```

Comment:
This comment is a bit misleading given that the consensus was that we should be giving an error. I'd write a slightly longer message recording the fact that it's uncertain and that we're going to come back to it.

Reply:
I will edit this to reflect Roman's suggested path forward.